### PR TITLE
Feature/widerruf second session

### PR DESCRIPTION
**Pull request by GeBon22**
Please do not regard the greyed out sections as I intend to use them in  the coming sprints. 
For further infos please take a look at the respective US. Cheers

### CODE REVIEW

File: pages/bookmarks/index.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
+/* import { set } from "mongoose"; */
```
Comment (stefankorder) [inline pages/bookmarks/index.js]: Please delete code you don't use anymore.

File: pages/bookmarks/index.js [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
 import styled from "styled-components";
+/* import { useState, useEffect } from "react"; */
```
Comment (stefankorder) [inline pages/bookmarks/index.js]: Please delete code you don't use anymore.

File: pages/bookmarks/index.js [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+
+  useEffect(() => {
+    if (isBookmarked.length === 0) {
+      setMarkedPage([]);
+    } else {
+      fetch(`${isBookmarked.join()}`)
+        .then((response) => response.json())
+        .then((markedPage) => setMarkedPage(markedPage));
+    }
+  }, [isBookmarked]); */
```
Comment (stefankorder) [inline pages/bookmarks/index.js]: Please delete code you don't use anymore.

File: pages/bookmarks/index.js [hunk 4]
```diff
@@ -1,12 +1,32 @@
...
+    <StyledBody>
+      <h3>This is the bookmarks page</h3>
+
+      {/*      {markedPage?.map((markedPage) => {
+        return (
+          <article key={markedPage.pathname}>
+            <h2>{markedPage.pathname}</h2>
+          </article>
+              );
+            })} */}
```
Comment (stefankorder) [inline pages/bookmarks/index.js]: Please delete code you don't use anymore.

File: pages/widerruf/frist.js [hunk 5]
```diff
@@ -10,6 +11,10 @@ export default function FristWiderruf() {
           lange kann ich widerrufen?
         </StyledBreadcrumb>
         <StyledArticle>
+          <StyledIcon
+            icon="mingcute:bookmark-fill"
+            color="#572887" /* onClick={handleToggle} */
```
Comment (stefankorder) [inline pages/widerruf/frist.js]: Please delete code you don't use anymore.

File: pages/widerruf/inhalt.js [hunk 6]
```diff
@@ -1,24 +1,72 @@
 import styled from "styled-components";
 import Link from "next/link";
-
+import {CopyToClipboard} from "react-copy-to-clipboard";
+import {Icon} from "@iconify/react";
+/* import { useState, useEffect } from "react"; */
+/* import { useRouter } from 'next/router';
+ */
```
Comment (stefankorder) [inline pages/widerruf/inhalt.js]: Please delete code you don't use anymore.

File: pages/widerruf/inhalt.js [hunk 7]
```diff
@@ -1,24 +1,72 @@
...
+  }, [isBookmarked]);
+
+
+
+const usePathname = () => {
+    const location = router.pathname
+    console.log(location);
+  }
+  usePathname(); */
+
```
Comment (stefankorder) [inline pages/widerruf/inhalt.js]: Please delete code you don't use anymore.

File: pages/widerruf/inhalt.js [hunk 8]
```diff
@@ -1,24 +1,72 @@
...
     <StyledBody>
       <StyledBreadcrumb>
         <StyledLink href="/widerruf">/Widerruf und Retoure</StyledLink>/Was muss
         in einem Widerruf stehen?
       </StyledBreadcrumb>
-      <h3>Was muss in einem Widerruf stehen?</h3>
       <StyledArticle>
+        <StyledIcon
+          icon="mingcute:bookmark-fill"
+          color="#572887" /* onClick={handleToggle} */
```
Comment (stefankorder) [inline pages/widerruf/inhalt.js]: Please delete code you don't use anymore.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Looks good so far! Please take a look at my comments and check where in your styling you can change px to relative units.

File: pages/bookmarks/index.js (hunk 3 above)
Comment (GeBon22) [reply pages/bookmarks/index.js]: thanks for your review. As mentioned above, I need the greyed out sections in my next step. If you dont mind, I will keep them.

Comment (stefankorder) [reply pages/bookmarks/index.js]: You should write your code you need for your next steps in another feature branch.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Take a look at my updated comment.

File: pages/bookmarks/index.js [hunk 9]
```diff
@@ -1,12 +1,10 @@
 import styled from "styled-components";
 
-export default function BookmarksPage() {
+export default function BookmarksPage(/* {isBookmarked} */) {
```
Comment (actyvystom) [inline pages/bookmarks/index.js]: one tiny peace of commented code left over, please remove ;)

**Review by actyvystom — COMMENTED** [review]
Good work, just one little peace of commented code left ;)

File: pages/bookmarks/index.js (hunk 9 above)
Comment (GeBon22) [reply pages/bookmarks/index.js]: resolved

**Pull request merged**